webpack.config.js

```diff
@@ -90,12 +91,12 @@ const config = {
                 presets: ['@babel/preset-env']
               }
             },
-            {
-              loader: 'eslint-loader',
-              options: {
-                fix: true
-              }
-            }
+            // {
```

Comment:
Why are we removing eslint?

Reply:
@woohsian is this a temporary removal?

Reply:
@arollason @ettiennegous Yes it is temporary